src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -54,6 +55,7 @@ public User update(@RequestBody User user, @PathVariable Long id) {
   }
 
   @PutMapping("/{userId}/books/add")
+  @ApiOperation(value= "Add a book to an user. The book must exist.")
   public User addBook(@RequestBody Book receivedBook, @PathVariable Long userId) {
```

Comment:
Add the possible HTTP status codes and their messages with the @ApiResponse annotation and a description of the parameters with the @ApiParam annotation